teradatalabs/mapr52-hive/files/supervisord.d/bootstrap.conf

```diff
@@ -0,0 +1 @@
+../../../../commons/supervisord.d/bootstrap.conf
```

Comment:
@sanjay990  new line  is not required as per artur , its same in other containers also